js/everything.js

```diff
@@ -22,9 +22,17 @@ function onYouTubeIframeAPIReady() {/* jshint ignore:line */
                 //console.log("State changed to " + event.data);
                 var playerState = event.data;
 
-                 if (playerState === YT.PlayerState.ENDED) {
-                    showPlayButton();
-                 }
+                switch (playerState) {
+                    case YT.PlayerState.ENDED:
```

Comment:
We can do slightly better here using fall-through so we don't have to duplicate code for `ENDED` and `PAUSED`. See this [SO question](http://stackoverflow.com/questions/13207927/switch-statement-multiple-cases-in-javascript#13207939) for an example.

Also, we should probably have a default condition while we're at at it.
We're currently not addressing these cases.
- `YT.PlayerState.BUFFERING`
- `YT.PlayerState.CUED`

Both of which I believe should have `showPlayButton();` as their default behavior. So, maybe we can simplify this whole thing quite a bit - just have 1 case for `YT.PlayerState.PLAYING`, and a default to `showPlayButton();`

Does that make sense?